scripts/subquery-e2e.py

```diff
@@ -121,3 +120,33 @@
 """
 Legacy Bridge Swap
 """
+url = "https://github.com/fetchai/fetch-ethereum-bridge-v1/releases/download/v0.2.0/bridge.wasm"
+contract_request = requests.get(url)
+
+if not os.path.exists("../.contract"):
+    os.mkdir("../.contract")
```

Comment:
We should probably exclude this directory from version control by adding it in a .gitignore.

Reply:
Should be fixed, thanks Bryan